ui/src/components/DialogComponents/TitleSection.tsx

```diff
@@ -34,6 +43,24 @@ const TitleSectionComponent: React.FC<Props> = ({
           <XIcon strokeWidth={"3"} width={"28"} />
         </IconButton>
       </Stack>
+      {additionalInfo && (
+        <Alert severity="info" className={styles["info"]}>
+          {additionalInfo.message}{" "}
+          {additionalInfo.documentation?.map((d) => (
+            <span>
+              <a
+                className={styles["doclink"]}
+                href={d?.url}
+                rel="noreferrer"
+                target="_blank"
+                key={d?.url}
+              >
+                {d?.text}
+              </a>{" "}
+            </span>
```

Comment:
The `key` prop needs to be at the top level of the mapped component. Also a key that is possibly undefined is a bad choice, but I think making the documentationLink non nullable in the schema will fix that.

```suggestion
            <span key={d.url}>
              <a
                className={styles["doclink"]}
                href={d?.url}
                rel="noreferrer"
                target="_blank"
              >
                {d?.text}
              </a>{" "}
            </span>
```